packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -419,9 +419,11 @@ class InspectorController extends DisposableController
 
   Future<void> _recomputeTreeRoot(
     RemoteDiagnosticsNode? newSelection, {
-    bool hideImplementationWidgets = false,
+    bool? hideImplementationWidgets,
   }) async {
     assert(!_disposed);
+    hideImplementationWidgets =
+        hideImplementationWidgets ?? _implementationWidgetsHidden.value;
```

Comment:
this can be:
`hideImplementationWidgets ??= _implementationWidgetsHidden.value;`

Reply:
Done!